Feature: Add option to remove the source_column_name on the union_relations macro (#624)
dbt-labs/dbt-utils: README.md

* `column_override` (optional): A dictionary of explicit column type overrides,
e.g. `{"some_field": "varchar(100)"}`.``
* `source_column_name` (optional, `default="_dbt_source_relation"`): The name of

> the column that records the source of this row. `source_column_name: none` can 
> be used to omit this column.

Reviewer: A bit concerned that using a colon might make people think that's how you pass the argument. (As I type this, I'm suddenly paranoid that you can use colons as well as equals signs 😱)

```suggestion
the column that records the source of this row. Pass `None` to omit this column from the results.
```
